packages/app-builder-lib/src/util/yarn.ts

```diff
@@ -104,8 +104,10 @@ function getPackageToolPath() {
   }
 }
 
-function isYarnPath(execPath: string | null | undefined) {
-  return process.env.FORCE_YARN === "true" || (execPath != null && path.basename(execPath).startsWith("yarn"))
+function isRunningYarn(execPath: string | null | undefined, userAgent: string | null | undefined) {
```

Comment:
Why not simply use process.env.npm_config_user_agent here to avoid it in clients?

Reply:
Good point. Will fix it shortly.